data/_models/shared/Collections.csv

```diff
@@ -1,53 +1,51 @@
 Collections,,design description,text,,,,,,,,,Short description of the study design of this cohort,DataCatalogueFlat,['Cohort'].includes(collectionType?.name)
 Collections,,design schematic,file,,,,,,,,,A schematic depiction of the study design of this cohort,DataCatalogueFlat,['Cohort'].includes(collectionType?.name)
 Collections,,collection type,ontology_array,,,CatalogueOntologies,Collection types,,,,,"The data collection type of this cohort, i.e. retrospective or prospective; if both, select both",DataCatalogueFlat,['Cohort'].includes(collectionType?.name)
-Collections,,logo,file,,,,,,,,,"Logo of the resource, for use on homepages etc.",DataCatalogueFlat,"['Cohort', 'Databank', 'Data source', 'Model', 'Network', 'Study'].includes(collectionType?.name)"
+Collections,,logo,file,,,,,,,,,"Logo of the resource, for use on homepages etc.",DataCatalogueFlat,"['Cohort', 'Databank', 'Data source', 'Model', 'Study'].includes(collectionType?.name)"
+Collections,,license,hyperlink,,,,,,,,dct:license,A legal document under which the resource is made available,DataCatalogueFlat,
 Collections,,population,heading,,,,,,,,,Description of the population that can potentially be captured in the resource,DataCatalogueFlat,"['Cohort', 'Databank', 'Data source'].includes(collectionType?.name)"
 Collections,,number of participants,int,,,,,,,,,Total number of individuals for which data is collected,"DataCatalogueFlat,EMA","['Cohort', 'Databank', 'Data source'].includes(collectionType?.name)"
 Collections,,number of participants with samples,int,,,,,,,,,Number of individuals for which samples are collected,DataCatalogueFlat,"['Cohort', 'Databank', 'Data source'].includes(collectionType?.name)"
 Collections,,underlying population,text,,,,,,,,,Provide a summary description of the underlying population (maximum 100 words) or URL to a description,DataCatalogueFlat,['Databank'].includes(collectionType?.name)
-Collections,,countries,ontology_array,,,CatalogueOntologies,Countries,,,,dcterms:spatial,Countries where data from this resource largely originate from,"DataCatalogueFlat,EMA","['Cohort', 'Databank', 'Data source', 'Model', 'Network', 'Study'].includes(collectionType?.name)"
-Collections,,regions,ontology_array,,,CatalogueOntologies,Regions,,,,dcterms:spatial,Geographical regions where data from this resource largely originate from,"DataCatalogueFlat,EMA","['Cohort', 'Databank', 'Data source'].includes(collectionType?.name)"
+Collections,,countries,ontology_array,,,CatalogueOntologies,Countries,,,,dct:spatial,Countries where data from this resource largely originate from,"DataCatalogueFlat,EMA","['Cohort', 'Databank', 'Data source', 'Model', 'Study'].includes(collectionType?.name)"
+Collections,,regions,ontology_array,,,CatalogueOntologies,Regions,,,,dct:spatial,Geographical regions where data from this resource largely originate from,"DataCatalogueFlat,EMA","['Cohort', 'Databank', 'Data source'].includes(collectionType?.name)"
```

Comment:
do we need both countries and regions here?

Reply:
They are used both in Cohorts tables, see: https://data-catalogue.molgeniscloud.org/catalogue/tables/#/Cohorts

Reply:
it is not a problem to have multiple coordinate systems. However we might want to prevent that both are used in same record.